indigo_app/templates/account/login.html

```diff
@@ -75,12 +75,15 @@ <h1 class="mb-3">Log In</h1>
           <i class="fab fa-fw fa-{{ provider.id }}"></i>
         {% endif %}
 
-        Log in with {{provider.name}}
+        {% blocktrans with provider=provider.name %}Log in with {{provider}}{% endblocktrans %}
       </a>
     {% endfor %}
   </div>
 </div>
 
-<div class="mt-3 text-muted text-center">By logging in you agree to our <a href="{% url 'terms_of_use' %}">Terms of Use &amp; Privacy Policy</a>.</div>
+<div class="mt-3 text-muted text-center">
+  {% url 'terms_of_use' as tos_url %}
+  {% blocktrans %}By logging in you agree to our <a href="{{ tos_url }}">Terms of Use and Privacy Policy</a>.{% endblocktrans %}
```

Comment:
no 'with tos_url=tos_url' needed after `blocktrans`? Would this be because of the use of `as` on 85?

Reply:
Yes. Blocktrans has in the surrounding context available to it, and `tos_url` is now part of the context.